### PR TITLE
Changes to build NIOE in dunfell

### PR DESCRIPTION
**Pull request by shruthi-ravi**
Workitems created to fix:

- [perf](https://ni.visualstudio.com/DevCentral/_workitems/edit/1105874)
- [florence](https://ni.visualstudio.com/DevCentral/_workitems/edit/1105791)

### CODE REVIEW

File: conf/distro/nilrt.inc [hunk 1]
```diff
@@ -21,6 +21,8 @@ DISTRO_FEATURES += "\
         ptest \
         selinux \
 	virtualization \
+	openstack \
+	OpenLDAP \
```
Comment (amstewart) [inline conf/distro/nilrt.inc]: What's with the indent differences here?

**Review by amstewart — CHANGES_REQUESTED** [review]
Could you add to your commit exactly what recipes we're enabling by adding these?

File: conf/distro/nilrt.inc (hunk 1 above)
Comment (shruthi-ravi) [reply conf/distro/nilrt.inc]: I'm dropping this commit altogether. The recipes were not enabled in sumo, and I'll make the changes to nsswitch for ldap in a separate commit.

File: recipes-bsp/grub/grub-nilrt.inc [hunk 2]
```diff
@@ -5,6 +5,7 @@ SRCREV = "832e7862089a7cbc985ae09a17f6827de6c85bf8"
 
 SRC_URI = "\
 	${NILRT_GIT}/grub.git;branch=${GRUB_BRANCH} \
+	file://x86-64-Treat-R_X86_64_PLT32-as-R_X86_64_PC32.patch \
```
Comment (amstewart) [inline recipes-bsp/grub/grub-nilrt.inc:8]: This is fine for now. Can you file a tech debt item to rebase our grub branch upon something more recent.

**Review by amstewart — APPROVED** [review]

File: recipes-bsp/grub/grub-nilrt.inc (hunk 2 above)
Comment (shruthi-ravi) [reply recipes-bsp/grub/grub-nilrt.inc:8]: Work [item](https://ni.visualstudio.com/DevCentral/_workitems/edit/1119988) created.

**Pull request merged**